src/components/Team/TeamList.js

```diff
@@ -1,15 +1,23 @@
 import React, { useEffect, useState } from "react";
 import PageTitle from "../Layout/PageTitle";
-import { Table, Divider, Tag, Card, Icon, Drawer, Popconfirm } from "antd";
+import { Table, Divider, Tag, Card, Icon, Popconfirm } from "antd";
 import "./style.css";
 import {
 	getUsersService,
 	toggleWebsiteSeen,
-	getRole
+	getRole,
+	toggleUserRevoke
 } from "../../utils/services";
 import { _notification } from "../../utils/_helpers";
 import UserOptions from "./UserOptions";
 import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+const StyledTable = styled(Table)`
+	.websiteShow {
+		display: ${props => (props.role === "lead" ? "block" : "none")};
+	}
+`;
 
 export default props => {
```

Comment:
I suggest using named functions instead of anonymous functions for easier debugging.

Reply:
Yes, this PR is in progress and I am using the named function now.